client/src/app/pages/login/login.page.scss

```diff
@@ -1,173 +1,221 @@
+.superior {
+  width:100%;
+  position:absolute;
+  /*top:0px;*/
+  display:grid;
+  justify-items:center;
 
-.container, ion-col {
-  height: 100%;
 }
 
-.container {
-  height: 100vh;
-  background-color: #f4eedb;
+.contenedor {
+  width:900px;
+  /*top:0px;*/
+  background-color:#F4EDDB;
+}
 
-  @media screen and (min-width: 720px) {
-    margin-left: 10%;
-    margin-right: 10%;
-  }
+.top {
+  background-repeat: no-repeat;
+  background-size: 900px 160px;
+  width:900px;
+  height:160px;
+  padding:0;
+  background-image:url('/assets/splash/header-rootrpg.png');    
+}
 
-  @media screen and (max-width: 720px) {
-    height: 150vh;
-  }
+.logo {
+  background-repeat: no-repeat;
+  background-size: 470px 160px;
+  width:470px;
+  height:160px;
+  padding:0;
+  margin-top:-125px;
+  background-image:url('/assets/splash/logo-rootrpg.png');
+}
 
-  .headers {
-    padding: 0;
-
-    .headers-container {
-      position: relative;
-    }
-
-    ion-col {
-      padding: 0;
-    }
-
-    .big-header-img {
-      width: 100%;
-      max-height: 200px;
-    }
-
-    .big-logo-img-container {
-      width: 100%;
-      position: absolute;
-      display: flex;
-      justify-content: center;
-    }
-
-    .big-logo-img {
-      width: 75%;
-      max-height: 250px;
-      margin-top: -170px;
-
-      @media screen and (min-width: 720px) {
-        width: 50%;
-      }
-    }
-  }
+.contenido {
+  height:640px;
 
-  .login-container {
-    padding-top: 10%;
-    height: 100%;
-  }
+  display:flex;
+  flex-flow: column wrap;
+  justify-content: flex-start;
 
-  .foreward {
-    --background: transparent;
-    box-shadow: none;
-  }
+  grid-row-gap: 15px;
+  grid-column-gap: 30px;
+
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  padding-left:40px;
+}
+
+.botonamarillo {
+  cursor: pointer;
+  border-radius: 10px;
+  background-color:#a3801f;
+  color:#fff;
+  width:120px;
+  padding:11px;
+  font-family:'Luminari';
+  font-size:18px;
+  text-transform: uppercase;
+}
+
+.botongris {
+  cursor:pointer;
+  border-radius:10px;
+  background-color:#2e6969;
+  color:#fff;
+  width:120px;
+  padding:11px;
+  font-family:'Luminari';
+  font-size:18px;
+  text-transform: uppercase;
+}
```

Comment:
these two are almost exactly the same, abstract the class definition out so you can use the common properties (cursor, border-radius, color, width, padding, font family, font size, and text transform can *all* be shared instead of duplicated).